.github/workflows/ticker-daily-deploy.yml

```diff
@@ -0,0 +1,61 @@
+name: Ticker Daily Deploy
+
+on:
+  push:
+    paths:
+      - ticker/**
+  workflow_dispatch:
+  schedule:
+    - cron: "0 7 * * *"
+
+jobs:
+  deploy-ticker:
+    defaults:
+      run:
+        working-directory: ./ticker
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+          cache: "npm"
+          cache-dependency-path: ./ticker/package-lock.json
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Deploy
+        run: |
+          if npm run deploy |& grep -q "Contract deployed"; then
+            echo "Contract successfully deployed!"
+          else
+            echo "Failed to deploy contract ..."
+            exit 1
+          fi
+
+      - name: Autonomy test
```

Comment:
I added this specific deploy test since for this contract it might be interesting for the developer to see how to test an autonomous smart contract and related features.
Wdyt @gregLibert ?